src/components/navigation/components/DrawerNavigation/index.tsx

```
@@ -1,10 +1,11 @@
+// @ts-nocheck
```

ESLint

[error] Do not use "@ts-nocheck" because it alters compilation errors.

(@typescript-eslint/ban-ts-comment)